build/Publish.targets

```diff
@@ -15,12 +14,8 @@
 
   <!-- UploadToAzure target comes from Build Tools -->
   <Target Name="PublishArtifacts"
-           DependsOnTargets="GenerateVersionBadge;
-                             GatherItemsForPattern;
-                             GenerateChecksums;
-                             UploadArtifactsToAzure;
-                             UploadChecksumsToAzure;
-                             PublishCliVersionBadge" />
+           DependsOnTargets="GatherItemsForPattern;
+                             UploadArtifactsToAzure;" />
 
   <Target Name="GatherItemsForPattern">
     <ItemGroup>
```

Comment:
This condition should not be needed:
Condition=" '$(IsLinuxDistroSpecific)' != 'true' "

Reply:
Since we have only one leg for CLI on VSO, we should not condition the NuPkg publish on OS and Arch:
Condition=" '$(PUBLISH_NUPKG_TO_AZURE_BLOB)' != '' AND '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' "

Reply:
Will clean this up in my next one.